### DISCUSSION TITLE
Run ArchRBrowser as a shiny app on a shiny server

### DISCUSSION THREAD

**Discussion by bj-chen** [GreenleafLab/ArchR#1381] [Questions / Documentation]
Hi ArchR team,

I'd like to know how to run function `ArchRBrowser()` as a shiny app on a shiny server? Many thanks!

Bei Jun

**Comment by rcorces** [comment]
This is not currently possible but is something we are working on. In the meantime, you could check out https://github.com/EskelandLab/ShinyArchRUiO which was developed by Ankush Sharma. We didnt play any role in the development of that tool but it is quite nice and serves as the basis for what we are working on in collaboration with that team to make part of ArchR.

**Reply by bj-chen** [reply]
Hi Ryan @rcorces: thanks for the information, will take a look at ShinyArchRUiO.

Regards,
Bei Jun
